.github/workflows/python.yaml

```diff
@@ -9,36 +9,48 @@ jobs:
     name: 'code-check > style:isort'
     runs-on: 'ubuntu-latest'
     container:
-      image: 'ghcr.io/rios0rios0/pipelines/python:3.10-pdm-bullseye'
+      image: 'ghcr.io/fnk0c/pipelines/python:3.10-pdm-bullseye'
     steps:
-      - run: pdm run isort --check-only .
+      - uses: actions/checkout@v3
+      - run: |
+          pdm install -v
```

Comment:
Is that possible to reuse this line in all 3 jobs?

Reply:
I mean, the same line 3 times. Is that possible to re-use it somehow?

Reply:
Also, by creating a new file having the `pdm install -v` command would still require us to implement the uses. Therefore, we would keep having the same amount of line we have today